src/docfx/build/toc/TocBuilder.cs

```diff
@@ -85,7 +85,15 @@ public void Build(ErrorBuilder errors, FilePath file)
                     var output = _templateEngine.RunJavaScript("toc.json.js", JsonUtility.ToJObject(model));
                     _output.WriteJson(Path.ChangeExtension(outputPath, ".json"), output);
                 }
-                else
+
+                if (_config.OutputType == OutputType.Json)
+                {
+                    var outputModel = JsonUtility.ToJObject(model);
+                    outputModel["schema"] = "toc";
```

Comment:
Add a get only property to `TocModel` and init the value to `toc`, to line 79/85/98 all pick the `schema` property.

Reply:
Done.